run-image/Dockerfile

```diff
@@ -1,8 +1,10 @@
 ARG base_image=index.docker.io/paketobuildpacks/run-jammy-full:0.1.99
 FROM ${base_image}
 
-ARG user_id=1000
-ARG group_id=1000
+ARG cnb_uid=1001
+ARG cnb_gid=1001
+ENV CNB_USER_ID=${cnb_uid}
+ENV CNB_GROUP_ID=${cnb_gid}
```

Comment:
While I understand that the user id during the build phase is `1001`, we should keep user `1000:1000` for when we run the image.